### PR TITLE
Add tests for `computational_resources` module.

### LINKED ISSUE #447: Test computational_resources module

### PR DESCRIPTION
**Pull request by yakutovicha**
fixes #447

### CODE REVIEW

**Comment by danielhollas** [comment]
Just FYI there is this known bug for creating a new `localhost` computer. Not sure if you've run into it while testng

https://github.com/aiidalab/aiidalab-widgets-base/issues/417

I'll leave this review to @unkcpz since he's been working on this widget a lot.

**Comment by yakutovicha** [comment]
> Just FYI there is this known bug for creating a new `localhost` computer. Not sure if you've run into it while testng


I prefer to fix that bug in a follow-up PR.

**Review by unkcpz — CHANGES_REQUESTED** [review]
@yakutovicha thanks a lot for testing this, very helpful. Just one request, all are good to me.

File: tests/test_computational_resources.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+            "safe_interval": 10,
+            "use_login_shell": True,
+        },
+    }
+
+    widget.computer_setup = computer_setup
+    assert widget.on_setup_computer()
+
+    # Check that the computer is created.
+    computer = orm.load_computer("daint")
```
[the remark this answers is not in the document]
Comment (yakutovicha) [reply tests/test_computational_resources.py:103]: Good point, @unkcpz. But the current widget is simply not supporting this use case. It deserves a PR on its own. Is it ok if we do that as a follow-up?

[the remark this answers is not in the document]
Comment (unkcpz) [reply tests/test_computational_resources.py:103]: > I guess we don't want the visuals to raise an exception. They should rather print a message.

Indeed. :+1:

**Review by unkcpz — APPROVED** [review]
It is all good from my side, thanks again this is very important test!

**Pull request merged**